src/combination.jl

```diff
@@ -40,11 +43,37 @@ function fullCombinationParam(origP1::Array{Float64, 2}, origP2::Array{Float64,
     return fullparam
 end
 
+""" This function calculates cell number for parameter sets that are the result of Bliss on prog. rates. """
+function BlissModelComb(bliss_comb, g1avg, g2avg, id1, id2)
```

Comment:
What are `id1` and `id2`? It doesn't look like you ever use them.

Reply:
Edited.